databricks-iris/{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -3,9 +3,19 @@ requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
+requires-python = ">=3.9"
 name = "{{ cookiecutter.python_package }}"
 readme = "README.md"
-dynamic = ["dependencies", "version"]
+dynamic = ["version"]
+dependencies = [
+    "ipython>=8.10",
+    "jupyterlab>=3.0",
+    "notebook",
+    "kedro~={{ cookiecutter.kedro_version }}",
+    "kedro[jupyter]",
+    "kedro-datasets[spark, pandas, spark.SparkDataset, pandas.ParquetDataset]>=1.0",
```

Comment:
Let's also update this requirement to `>=3.0` like in all the other starters, to use the "new" way we define the dependency.